### ISSUE TITLE
Two tests of jupyterlab failed with 2.27.0

### ISSUE THREAD

**Issue by frenzymadness** [jupyterlab/jupyterlab_server#449]
I'm building jupyterlab 4.1.5 on top of the latest jupyterlab_server 2.27.0 as RPM packages for Fedora Linux and the following two tests failed:

```
=================================== FAILURES ===================================
_____________________ TestExtension.test_disable_extension _____________________

self = <jupyterlab.tests.test_jupyterlab.TestExtension testMethod=test_disable_extension>

    def test_disable_extension(self):
        options = AppOptions(app_dir=self.tempdir())
        assert install_extension(self.mock_extension, app_options=options) is True
>       assert disable_extension(self.pkg_names["extension"], app_options=options) is True

jupyterlab/tests/test_jupyterlab.py:556: 
_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ 
jupyterlab/commands.py:541: in disable_extension
    return handler.toggle_extension(extension, True, level=level)
jupyterlab/commands.py:1127: in toggle_extension
    write_page_config(page_config, level=level)
/usr/lib/python3.12/site-packages/jupyterlab_server/config.py:217: in write_page_config
    cm.set("page_config", page_config)  # type:ignore[no-untyped-call]
/usr/lib/python3.12/site-packages/jupyter_server/services/config/manager.py:35: in set
    return self.write_config_manager.set(section_name, data)
/usr/lib/python3.12/site-packages/jupyter_server/config_manager.py:112: in set
    self.ensure_config_dir_exists()
/usr/lib/python3.12/site-packages/jupyter_server/config_manager.py:66: in ensure_config_dir_exists
    os.makedirs(self.config_dir, 0o755)
<frozen os>:215: in makedirs
    ???
<frozen os>:215: in makedirs
    ???
_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ 

name = '/usr/etc', mode = 511, exist_ok = False

>   ???
E   PermissionError: [Errno 13] Permission denied: '/usr/etc'

<frozen os>:225: PermissionError
----------------------------- Captured stdout call -----------------------------
-\|/-\|
_____________________ TestExtension.test_enable_extension ______________________

self = <jupyterlab.tests.test_jupyterlab.TestExtension testMethod=test_enable_extension>

    def test_enable_extension(self):
        options = AppOptions(app_dir=self.tempdir())
        assert install_extension(self.mock_extension, app_options=options) is True
>       assert disable_extension(self.pkg_names["extension"], app_options=options) is True

jupyterlab/tests/test_jupyterlab.py:576: 
_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ 
jupyterlab/commands.py:541: in disable_extension
    return handler.toggle_extension(extension, True, level=level)
jupyterlab/commands.py:1127: in toggle_extension
    write_page_config(page_config, level=level)
/usr/lib/python3.12/site-packages/jupyterlab_server/config.py:217: in write_page_config
    cm.set("page_config", page_config)  # type:ignore[no-untyped-call]
/usr/lib/python3.12/site-packages/jupyter_server/services/config/manager.py:35: in set
    return self.write_config_manager.set(section_name, data)
/usr/lib/python3.12/site-packages/jupyter_server/config_manager.py:112: in set
    self.ensure_config_dir_exists()
/usr/lib/python3.12/site-packages/jupyter_server/config_manager.py:66: in ensure_config_dir_exists
    os.makedirs(self.config_dir, 0o755)
<frozen os>:215: in makedirs
    ???
<frozen os>:215: in makedirs
    ???
_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ 

name = '/usr/etc', mode = 511, exist_ok = False

>   ???
E   PermissionError: [Errno 13] Permission denied: '/usr/etc'

<frozen os>:225: PermissionError
----------------------------- Captured stdout call -----------------------------
-\|/-\|
=============================== warnings summary ===============================
../../../../usr/lib/python3.12/site-packages/pytest_jupyter/jupyter_server.py:17
  /usr/lib/python3.12/site-packages/pytest_jupyter/jupyter_server.py:17: DeprecationWarning: Jupyter is migrating its paths to use standard platformdirs
  given by the platformdirs library.  To remove this warning and
  see the appropriate new directories, set the environment variable
  `JUPYTER_PLATFORM_DIRS=1` and then run `jupyter --paths`.
  The use of platformdirs will be the default in `jupyter_core` v6
    import jupyter_core.paths

../../../../usr/lib/python3.12/site-packages/_pytest/config/__init__.py:1373
  /usr/lib/python3.12/site-packages/_pytest/config/__init__.py:1373: PytestConfigWarning: Unknown config option: ignore
  
    self._warn_or_fail_if_strict(f"Unknown config option: {key}\n")

-- Docs: https://docs.pytest.org/en/stable/how-to/capture-warnings.html
----- generated xml file: /builddir/build/BUILD/jupyterlab-4.1.5/junit.xml -----
=========================== short test summary info ============================
FAILED jupyterlab/tests/test_jupyterlab.py::TestExtension::test_disable_extension
FAILED jupyterlab/tests/test_jupyterlab.py::TestExtension::test_enable_extension
====== 2 failed, 64 passed, 14 deselected, 2 warnings in 64.44s (0:01:04) ======
```

I admit that the build environment is kinda special and I don't know yet what is wrong but it seems to be related to the latest changes here.

**Comment by jtpio** [comment]
Thanks @frenzymadness for reporting :+1: 

Yes it seems related to the changes in https://github.com/jupyterlab/jupyterlab_server/pull/448.

Linking to https://github.com/jupyterlab/jupyterlab/issues/16222 as related.

**Comment by frenzymadness** [comment]
Thank you. It seems to be fixed in 2.27.1.
